Feat/issue 271 rounding digits (#272)
AlphaGenes/AlphaPeel: docs/source/usage.rst

      -out_id_only          Suppress output for individuals not present in
                            the file specified with -out_id_order. It also suppresses
                            "dummy" individuals.

>       -out_digits           Specify the number of digits to round to in the outputs except ``alt_allele_prob``, ``geno_error_prob`` and ``seq_error_prob``. Default: 4.

Reviewer: ```suggestion
      -out_digits           Specify the number of digits to round the outputs.
                            Does not apply to outputs from ``alt_allele_prob``, 
                            ``geno_error_prob``, and ``seq_error_prob``.
                            Default: 4.
```
